app/views/users/show.html.erb

```diff
@@ -1,2 +1,32 @@
 <h1>Specific user</h1>
-<p>List of specific users</p>
\ No newline at end of file
+
+<div class="user-card">
+  <%= image_tag(@user.photo, alt: @user.name, class: "user-photo") %>
+  <div class="user-details">
+    <h2><%= @user.name %></h2>
+    <p>Number of posts: <%= @user.posts.count %></p>
+  </div>
+</div>
+
+<div class="user-bio">
+  <h2>Bio</h2>
+  <p>
+    <%= @user.bio %>
+  </p>
+</div>
+
+<div class="user-posts">
+  <% @user.posts.limit(3).each do |post| %>
+    <div class="post">
+      <h2><%= link_to "#{post.title}", user_post_path(post.author, post) %></h2>
+      <p><%= post.text %></p>
+      <p class="counter">Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></p>
+    </div>
+  <% end %>
```

Comment:
* [ ]  Please make sure to use the `most_recent_posts` method you created in the last milstone in your User model to render the 3 most recent posts as mentioned in the project requirements. Do this in your user show page.
```ruby
# app/views/users/show.html.erb

[...]

<% @user.recent_post.each do |post| %>
```

Reply:
Hello @BenMukebo I just added the changes and it's working. 
Kindly review for me.